### PR TITLE
[14.0][MIG] sale_by_packaging: migration to 14.0

### PR DESCRIPTION
**Pull request by kevinkhao**
Supersedes https://github.com/OCA/sale-workflow/pull/1398
Integrates https://github.com/OCA/sale-workflow/pull/1276 https://github.com/OCA/sale-workflow/pull/1375

- [x] depend on https://github.com/OCA/sale-workflow/pull/1396

### CODE REVIEW

File: sale_by_packaging/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale By Packaging",
     "summary": "Manage sale of packaging",
-    "version": "13.0.1.4.0",
+    "version": "14.0.0.1.0",
```
Comment (simahawk) [inline sale_by_packaging/__manifest__.py]: version seems wrong

File: sale_by_packaging/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
 * Akim Juillerat <akim.juillerat@camptocamp.com>
 * Thomas Nowicki <thomas.nowicki@camptocamp.com>
 * François Honoré <francois.honore@acsone.eu>
+* Hiep (Nguyen Hoang) <hiepnh@trobz.com>
```
Comment (simahawk) [inline sale_by_packaging/readme/CONTRIBUTORS.rst:4]: this is a straight migration, I don't think we add contribs here in such cases

File: test-requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-product-packaging-type @ git+https://github.com/trobz/product-attribute@14.0-mig-product_packaging_type#subdirectory=setup/product_packaging_type
```
Comment (simahawk) [inline test-requirements.txt]: remember to drop these

**Review by simahawk — CHANGES_REQUESTED** [review]
@kevinkhao could you link depending PRs in the description?
Also, I think the migration commit should stay at the end since the integrated work was merged before and you might have changes to be done about them

**Comment by hparfr** [comment]
@kevinkhao can you please update your PR?

**Comment by simahawk** [comment]
@kevinkhao ping

File: sale_by_packaging/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (kevinkhao) [reply sale_by_packaging/readme/CONTRIBUTORS.rst:4]: I keep it for now following the discussion here https://odoo-community.org/groups/contributors-15/contributors-200441?mode=thread&date_begin=&date_end=
I don't care to add myself but maybe that guy cares, better be too generous than the opposite IMHO

**Comment by kevinkhao** [comment]
@simahawk thanks for feedback
@hparfr done

File: sale_by_packaging/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (simahawk) [reply sale_by_packaging/readme/CONTRIBUTORS.rst:4]: yeah that's fine, I rolled back :wink:

File: test-requirements.txt (hunk 3 above)
Comment (simahawk) [reply test-requirements.txt]: :arrow_up:

Comment (kevinkhao) [reply test-requirements.txt]: sorry, missed that, done

**Comment by sebastienbeau** [comment]
@simahawk @ivantodorovich shoudl be ready for merge, just need to wait for travis

**Comment by sebastienbeau** [comment]
Green !

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@hparfr can you review?

**Review by hparfr — APPROVED** [review]
LGTM

**Pull request merged**